substrate/primitives/consensus/slots/src/lib.rs

```diff
@@ -91,6 +91,11 @@ impl Slot {
 		Slot(timestamp.as_millis() / slot_duration.as_millis())
 	}
 
+	/// Timestamp of the start of the slot. Returns None if would overflow for given SlotDuration.
```

Comment:
```suggestion
	/// Timestamp of the start of the slot.
	///
	/// Returns `None` if would overflow for given `SlotDuration`.
```

Reply:
@LGLO please apply and then we can merge this.